servant-dhall.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      >=1.10
 name:               servant-dhall
-version:            0.3
+version:            0.3.0.1
```

Comment:
The interface changes, so it has to be minor bump at least.

`servant-dhall` follows https://pvp.haskell.org/

> *Non-breaking change*. Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then A.B MAY remain the same but the new C **MUST** be greater than the old C. Note that modifying imports or depending on a newer version of another package may cause extra non-orphan instances to be exported and thus force a minor version change.

Reply:
Good point. Changed to 0.3.1.